.gitignore

```diff
@@ -1,2 +1,4 @@
 .venv
 docs
+eos-data/
+mariadb-data/
```

Comment:
Ein einziges `data`-Verzeichnis mit Unterverzeichnissen wäre gut, damit es im Projekt-Root nicht zu viele Einträge gibt.

Reply:
mariadb ist eine dependency, die daten sollten wir nicht mit applikationsdaten vermischen.

Reply:
`eos-data` macht momentan nichts, sollte nur ein Beispiel sein.
Prinzipiell hätte ich die mounts lieber alle in ein Verzeichnis gepackt, bedeutet dann aber, dass dieses Verzeichnis erst erstellt, oder mit `.gitkeep` eingecheckt werden muss.